lib/templates.js

```diff
@@ -1,81 +1,112 @@
 'use strict';
 
-var api = 'templates'
-  , Promise = require('./Promise')
-  , toApiFormat = require('./toApiFormat');
+const api = 'templates';
+const Promise = require('./Promise');
```

Comment:
What others? They are all moving to const I believe. You just may not see it in this branch. I just updated this branch from wip-2.0.0 so you might actually see it now.